spec/integration/grape_spec.rb

```diff
@@ -27,18 +30,24 @@ class GrapeTestApp < ::Grape::API
           end
         end
       end
+    end
 
-      config = { api_request_time: '100ms' }
-      ElasticAPM::Grape.start(GrapeTestApp, config)
+    before do
+      MockIntake.instance.stub!
+      ElasticAPM::Grape.start(GrapeTestApp, { api_request_time: '100ms' })
```

Comment:
`with_agent` takes a block so it's hard to get into a `before :all`.

Reply:
Could we use `around` then?

Reply:
`around` are called before `before`/`after` and so our http mocking efforts aren't set up in time. This can be solved with manually stubbing (like here) so, yeah, I guess we could.

Reply:
I don't want to over-complicate this, just noticed this was inconsistent with the other tests. It's up to you whether you want to leave it = )